meta-resin-common/recipes-core/systemd/systemd/coredump.conf

```diff
@@ -0,0 +1,2 @@
+[Coredump]
+Storage=none
```

Comment:
If we are mapping both directories to the same persistent partition does it matter?  I guess we can then rely on the journald log size restrictions to not fill up the storage.